src/main/java/org/folio/verticle/consumers/consumerstorage/KafkaConsumersStorage.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.verticle.consumers.consumerstorage;
+
+import org.folio.kafka.KafkaConsumerWrapper;
+
+import java.util.Collection;
+
+/**
+ * Storage for Kafka consumers. Consumers are not tenant specific.
+ */
+public interface KafkaConsumersStorage {
```

Comment:
Consider adding this to folio-kafka-wrapper so that SRM and mod-orders use it.

Reply:
This feature has now been removed from mod-orders because it is used for flow control - this functionality is not needed in mod-orders now.